accounts-db/accounts-hash-cache-tool/src/main.rs

```diff
@@ -143,6 +144,20 @@ fn main() {
                                      Must be a power of two, greater than 0, \
                                      and less-than-or-equal-to 16,777,216 (2^24)"
                                 ),
+                        )
+                        .arg(
+                            Arg::with_name("bin_of_interest")
```

Comment:
This works. 
I wonder if we could relax the single-bin to a bin-range? 
For 8192 bins, we would need to run the command and scan the cache 8192 times to check all bins.

Reply:
lgtm.